infra/cifuzz/actions/run_fuzzers/action.yml

```diff
@@ -44,3 +44,4 @@ runs:
     # for running because we use it to distinguish OSS-Fuzz from non-OSS-Fuzz.
     # We should do something explicit instead.
     BUILD_INTEGRATION_PATH: ${{ inputs.build-integration-path }}
+    LOW_DISK_SPACE: '1'
```

Comment:
Why is this a string attribute, why not bool like true or false ? just like required attributes above.

Reply:
Maybe LOW_DISK_SPACE_CLEANUP for clarity?

Reply:
What's the downside to always cleaning up? Can we just always have this behaviour on without adding a flag?

Reply:
> What's the downside to always cleaning up? Can we just always have this behaviour on without adding a flag?

The downside is it will make things hard to debug. Maybe it should be opt-out. I'll probably change this in another PR since this PR is a bit urgent because zstd's CI is breaking without it.

Reply:
> Maybe LOW_DISK_SPACE_CLEANUP for clarity?

I think the current name.

Reply:
> Why is this a string attribute, why not bool like true or false ? just like required attributes above.

Done.